src/stable/test/benchmarks/actor.ts

```diff
@@ -1,6 +1,6 @@
 import { Actor, ActorMethod, ActorSubclass } from '@dfinity/agent';
 
-import { createAuthenticatedAgent } from '#utils/dfx';
+import { createAuthenticatedAgent } from '#experimental/utils/dfx';
```

Comment:
on the one hand it's testing and benchmark stuff... on the other hand it's in the src dir